### PR TITLE
Fix and move az-tests

### PR DESCRIPTION
**Pull request by pchampin**

### CODE REVIEW

**Review by afs — APPROVED** [review]

**Review by Tpt — APPROVED** [review]

File: rdf/rdf11/rdf-mt/az-tests/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is an alternative test suite for RDF 1.1 (and RDF 1.2) Semantics.
+It is not part of the Working Group's endorsed test suite,
+as it tests corner-cases that have of limited practical interest,
```
Comment (TallTed) [inline rdf/rdf11/rdf-mt/az-tests/README.txt]:
```suggestion
as it tests corner-cases that are of limited practical interest,
```
